test/e2e/specs/site-editor/patterns.spec.js

```diff
@@ -1,252 +0,0 @@
-				wp_pattern_category: [],
-			} ),
-			requestUtils.createBlock( {
-				title: 'Synced footer',
-				status: 'publish',
-				content: `<!-- wp:paragraph -->\n<p>Synced footer</p>\n<!-- /wp:paragraph -->`,
-				wp_pattern_category: [],
-			} ),
-		] );
-
-		await admin.visitSiteEditor( { path: '/patterns' } );
-
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 3 );
-
-		await patterns.content
-			.getByRole( 'searchbox', { name: 'Search patterns' } )
-			.fill( 'footer' );
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 2 );
-		expect(
-			await patterns.list
-				.getByRole( 'listitem' )
-				.getByRole( 'heading' )
-				.allInnerTexts()
-		).toEqual(
-			expect.arrayContaining( [ 'Unsynced footer', 'Synced footer' ] )
-		);
-
-		const searchBox = patterns.content.getByRole( 'searchbox', {
-			name: 'Search patterns',
-		} );
-
-		await searchBox.fill( 'no match' );
-		await expect( patterns.content ).toContainText( 'No patterns found.' );
-
-		await patterns.content
-			.getByRole( 'button', { name: 'Reset search' } )
-			.click();
-		await expect( searchBox ).toHaveValue( '' );
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 3 );
-
-		const syncFilter = patterns.content.getByRole( 'radiogroup', {
-			name: 'Filter by sync status',
-		} );
-		await expect(
-			syncFilter.getByRole( 'radio', { name: 'All' } )
-		).toBeChecked();
-
-		await syncFilter
-			.getByRole( 'radio', { name: 'Synced', exact: true } )
-			.click();
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 1 );
-		await expect( patterns.list.getByRole( 'listitem' ) ).toContainText(
-			'Synced footer'
-		);
-
-		await syncFilter.getByRole( 'radio', { name: 'Not synced' } ).click();
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 2 );
-		expect(
-			await patterns.list
-				.getByRole( 'listitem' )
-				.getByRole( 'heading' )
-				.allInnerTexts()
-		).toEqual(
-			expect.arrayContaining( [ 'Unsynced header', 'Unsynced footer' ] )
-		);
-
-		await searchBox.fill( 'footer' );
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 1 );
-		await expect( patterns.list.getByRole( 'listitem' ) ).toContainText(
-			'Unsynced footer'
-		);
-
-		await syncFilter.getByRole( 'radio', { name: 'All' } ).click();
-		await expect( patterns.list.getByRole( 'listitem' ) ).toHaveCount( 2 );
-		expect(
-			await patterns.list
-				.getByRole( 'listitem' )
-				.getByRole( 'heading' )
-				.allInnerTexts()
-		).toEqual(
-			expect.arrayContaining( [ 'Unsynced footer', 'Synced footer' ] )
-		);
-	} );
-} );
-
-class Patterns {
-	/** @type {import('@playwright/test').Page} */
-	#page;
-
-	constructor( { page } ) {
-		this.#page = page;
-
-		this.content = this.#page.getByRole( 'region', {
-			name: 'Patterns content',
-		} );
-		this.navigation = this.#page.getByRole( 'region', {
-			name: 'Navigation',
-		} );
-		this.list = this.content.getByRole( 'list' );
-	}
-}
```

Comment:
Feels like there's an opportunity to test our new screens better. Maybe we should adapt the tests instead?

Reply:
Yeah, we need to add tests for every new page. I am unsure if I'll have time for that before RC, but can commit to do it in a follow-up.

Reply:
I agree we should adapt the tests. If the plan is to do it a follow up, maybe just skip them for now?

Reply:
Sure, I can skip them https://github.com/WordPress/gutenberg/pull/58139/commits/4506147debca570cd989b6a13466e126ea1fb205 Though I'm skeptical we can reuse any of that code.

Reply:
I added tasks for adding e2e test for every page in the tracking issue.